daml-lf/transaction/src/main/scala/com/daml/lf/Error.scala

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.lf
+package interpretation
+
+import com.daml.lf.data.Ref.{Location, Party, TypeConName}
+import com.daml.lf.transaction.{GlobalKey, NodeId}
+import com.daml.lf.language.Ast
+import com.daml.lf.value.Value
+import com.daml.lf.value.Value.ContractId
+
+/** Daml exceptions that should be reported to the user
+  */
+sealed abstract class Error extends Serializable with Product {
+  override def toString: String = s"$productPrefix(${productIterator.mkString("\n")}"
```

Comment:
```suggestion
  override def toString: String = s"$productPrefix(${productIterator.mkString(System.lineSeparator())}"
```

Minor, useful if printed on Windows.